ProGetAutomation/Functions/Invoke-ProGetRestMethod.ps1

```diff
@@ -6,58 +6,59 @@ function Invoke-ProGetRestMethod
-        [object]
-        # A session object that represents the ProGet instance to use. Use the `New-ProGetSession` function to create session objects.
-        $Session,
+        [object] $Session,
 
-        [Parameter(Mandatory)]
-        [String]
         # The path to the API endpoint.
-        $Path,
+        [Parameter(Mandatory)]
+        [String] $Path,
 
-        [Microsoft.PowerShell.Commands.WebRequestMethod]
         # The HTTP/web method to use. The default is `POST`.
-        $Method = [Microsoft.PowerShell.Commands.WebRequestMethod]::Post,
+        [WebRequestMethod] $Method = [WebRequestMethod]::Post,
 
-        [Parameter(ParameterSetName='ByParameter')]
-        [hashtable]
         # The parameters to pass to the method.
-        $Parameter,
+        [Parameter(ParameterSetName='ByParameter')]
+        [hashtable] $Parameter,
 
+        # Controls how the parameters are sent to the API. The default is `Form`, which sends them as URL-encoded
+        # name/value pairs (i.e. like a HTML form submission). The other options is `Json`, which converts the
+        # parameters to JSON and sends that JSON text as the content/body of the request. This parameter is ignored if
+        # there are no parmaeters to send or if the `InFile` parameter is used.
         [Parameter(ParameterSetName='ByParameter')]
-        [string]
         [ValidateSet('Form','Json')]
-        # Controls how the parameters are sent to the API. The default is `Form`, which sends them as URL-encoded name/value pairs (i.e. like a HTML form submission). The other options is `Json`, which converts the parameters to JSON and sends that JSON text as the content/body of the request. This parameter is ignored if there are no parmaeters to send or if the `InFile` parameter is used.
-        $ContentType,
+        [String] $ContentType,
 
-        [Parameter(ParameterSetName='ByFile')]
-        [String]
         # Send the contents of the file at this path as the body of the web request.
-        $InFile,
+        [Parameter(ParameterSetName='ByFile')]
+        [String] $InFile,
 
-        [Parameter(ParameterSetName='ByContent')]
-        [String]
         # Send the content of this string as the body of the web request.
-        $Body,
+        [Parameter(ParameterSetName='ByContent')]
+        [String] $Body,
 
-        [Switch]
-        # Return the raw content from the request instead of attempting to convert the response from JSON into an object.
-        $Raw
+        # Return the raw content from the request instead of attempting to convert the response from JSON into an
+        # object.
+        [switch] $Raw
     )
 
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
     $uri = New-Object 'Uri' -ArgumentList $Session.Url,$Path
 
-    $requestContentType = 'application/json; charset=utf-8'
+    $requestContentType = 'application/json' #; charset=utf-8'

```

Comment:
Intentional comment or debugging artifact?

Reply:
API was throwing exceptions if the charset directive was present. I reported it as a bug ,with the intent to uncomment when it was fixed, but everything works with it missing, so I'll remove.